src/part-5-intro.md

```diff
@@ -7,13 +7,14 @@ changes.
 It's often useful to think of compilers as being composed of a _frontend_ and a
 _backend_  (though in rustc, there's not a sharp line between frontend and
 backend). The _frontend_ is responsible for taking raw source code, checking it
```

Comment:
We can still avoid *front* and *back* terms, and one benefit is we don't have to talk about "sharp lines".

Reply:
I tried to de-emphasize it and disambiguate it from "codegen backend". Let me know what you think.

Reply:
I like